RevenueCatUI/CustomerCenter/Data/SubscriptionInformation.swift

```diff
@@ -0,0 +1,24 @@
+//
+//  SubscriptionInformation.swift
+//
+//
+//  Created by Cesar de la Vega on 28/5/24.
+//
+
+import Foundation
+
+struct SubscriptionInformation {
+
+    let title: String
+    let duration: String
+    let price: String
+    let nextRenewal: String
```

Comment:
You're correct, I am going to test deserializing and making these types more semantically correct. Will probably open another PR to keep this one simple and more focus on the view

Reply:
actually, updating them now, since in the other PR, I've decided to create a response object. thanks for pointing this out!

Reply:
oh lol this is SubscriptionInformation, which doesn't come from the backend lol, will definitely update these

Reply:
ok, gave it another thought, since this is only used as a data class to hold information that is created in the view model I think it's better to leave it as is. I renamed some properties so they better represent what they hold.